twitivity.py

```diff
@@ -63,6 +63,15 @@ def register_webhook(self, callback_url: str) -> json:
         except Exception:
             raise
 
+    def delete_webhook(self, webhook_id: int) -> json:
```

Comment:
Is the webhook_id being passed to the API endpoint?

Reply:
Force pushed the fix.
Thanks for the review!